yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/CameraGraphicView.kt

```diff
@@ -12,45 +12,63 @@
 package ai.cyberlabs.yoonit.camera
 
 import ai.cyberlabs.yoonit.camera.utils.BlurBuilder
-import ai.cyberlabs.yoonit.camera.utils.crop
-import ai.cyberlabs.yoonit.camera.utils.mirror
-import ai.cyberlabs.yoonit.camera.utils.rotate
 import android.content.Context
 import android.graphics.*
 import android.util.AttributeSet
 import android.view.View
 
 /**
- * Custom view to bind face bounding box to the preview view.
+ * Custom view to draw face detection box and face blurred.
  */
 class CameraGraphicView constructor(
     context: Context,
     attrs: AttributeSet? = null
 ) : View(context, attrs) {
 
-    private var boundingBox: RectF? = null
+    // The face detection box within the graphic view.
+    private var faceDetectionBox: RectF? = null
+        set(value) {
+            this.drawFaceDetectionBox = value != null
+            field = value
+        }
+    private var drawFaceDetectionBox: Boolean = false
 
-    private var imageBitmap: Bitmap? = null
+    // The face bitmap.
+    private var faceBlurBitmap: Bitmap? = null
+        set(value) {
+            this.drawFaceBlurBitmap = value != null
+            field = value
+        }
+    private var drawFaceBlurBitmap: Boolean = false
 
     override fun draw(canvas: Canvas) {
         super.draw(canvas)
 
-        boundingBox?.let {
-
-            imageBitmap?.let {bitmap ->
-                canvas.drawBitmap(BlurBuilder.blur(context, bitmap), null, it, null)
-            }
+        if (this.faceBlurBitmap != null && this.drawFaceBlurBitmap) {
```

Comment:
Let call normally is the pattern to perform null safe operations in kotlin, why remove let call?